src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1224,7 +1224,9 @@ Color.white, new Procedure("condition4"),
 			item.inventorySize = 10;
 			item.inventoryStackSize = 42;
 			item.guiBoundTo = emptyLists || guis.isEmpty() ? null : getRandomItem(random, guis);
-            item.openGUIOnRightClick = new LogicProcedure(item.guiBoundTo == null ? null : "condition3", _true);
+			if (item.guiBoundTo != null) {

```

Comment:
Don't think you need this as even if guiBoundTo is null, you can still pass this parameter. Similar to how UI component keep value and the value is saved even if it is disabled

Reply:
Alright, will keep future conversations unresolved 👍